runtime/doc/options.txt

```diff
@@ -6501,7 +6501,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	tags file are relative to the directory where the tags file is.
 
 						*'tags'* *'tag'* *E433*
-'tags' 'tag'		string	(default "./tags,tags", when compiled with
+'tags' 'tag'		string	(default "./tags;,tags", when compiled with
 				|+emacs_tags|: "./tags,./TAGS,tags,TAGS")
```

Comment:
Actually, nah. There's not many references, so they can all be removed here IMO.